src/snowflake/cli/api/project/schemas/entities/common.py

```diff
@@ -59,6 +61,27 @@ def get_type(cls) -> str:
         return cls.model_fields["type"].annotation.__args__[0]
 
     meta: Optional[MetaField] = Field(title="Meta fields", default=None)
+    identifier: Optional[Union[Identifier | str]] = Field(
+        title="Entity identifier", default=None
+    )
+    _key: Optional[str] = PrivateAttr(default=None)
+
+    def set_key(self, value: str):
```

Comment:
can we add docstring to this or to the key attribute?

Reply:
Added and renamed this to `entitiy_id`